### PR TITLE
Store Home Fragment post objectId : score pairings in SharedPreferences.

### PR DESCRIPTION
**Pull request by kellyyyang**

### CODE REVIEW

File: app/src/main/java/com/codepath/travelbud/parse_classes/Post.java [hunk 1]
```diff
@@ -25,6 +32,17 @@ public class Post extends ParseObject {
     public static final String KEY_HASHTAGS = "hashtags";
     public static final String KEY_VISIBILITY = "visibility";
 
+//    @ColumnInfo(name = "description")
+//    private String description;
+//
+//    @ColumnInfo(name = "location_string")
+//    private String location_string;
+//
```
Comment (JinyueHu) [inline app/src/main/java/com/codepath/travelbud/parse_classes/Post.java:40]: Hey Kelly, I noticed these filed are no longer used, do we want to keep them in master? Maybe we could delete them?

File: app/src/main/java/com/codepath/travelbud/parse_classes/Post.java [hunk 2]
```diff
@@ -15,6 +19,9 @@
 @ParseClassName("Post")
 public class Post extends ParseObject {
 
+//    @PrimaryKey(autoGenerate = true)
```
Comment (JinyueHu) [inline app/src/main/java/com/codepath/travelbud/parse_classes/Post.java:22]: Same here :)

**Review by JinyueHu — COMMENTED** [review]
In order to make the code clean, we could remove the unused code and comments?

File: app/src/main/java/com/codepath/travelbud/parse_classes/Post.java (hunk 1 above)
Comment (kellyyyang) [reply app/src/main/java/com/codepath/travelbud/parse_classes/Post.java:40]: Yes, I'll delete them!

**Pull request merged**